_source/_posts/2022-01-04-angular-auth0-quickly.md

```diff
@@ -0,0 +1,177 @@
+---
+layout: blog_post
+title: "Add OpenID Connect to Angular Apps Quickly"
+author: matt-raible
+by: advocate
+communities: [javascript]
+description: "This tutorial shows you how to add OpenID Connect authentication with Auth0 in just a few minutes."
+tags: [javascript, angular, typescript, auth0]
+tweets:
+- ""
+- ""
+- ""
+image:
+type: conversion
+github: https://github.com/oktadev/auth0-angular-example
+---
```

Comment:
My  primary suggestions are:

- Run through the CLI commands one more time in a fresh directory. This may be just my machine but the default scaffolded app was out of date, which was strange. Node 16.3.1, npm 8.3.
- I may reorder the main content. I would have "What we're going to do in this demo" before an Angular intro/retrospective.
- Reorder the app registration sections to allow CLI or UI. Perhaps make a note to save the client information etc.
- I would add a bit more detail about what's actually happening for a couple of the steps. I've made some suggestions in that regard.

Reply:
Thanks for the review @Ryanman. I recorded a screencast of this post and didn't experience any issues back in December. If I make any of the above changes, do you think I'll need to re-record the screencast? Here's a link to it (you might need to request permission to view it). https://drive.google.com/file/d/1kyQdVbuAOMc0YuD0Bo55sOLsPmJ74Sli/view?usp=sharing